setup.py

```diff
@@ -24,8 +24,13 @@
         "Framework :: Django",
     ],
     install_requires=[
-        'Django >= 1.6',
+        'Django>=1.6',
         'Unidecode>=0.04.17',
         'pyprind==2.9.9',
+        'django-is-core>=1.3.145',
```

Comment:
Je to logiké, protože v `chamber/multidomains/auth/middleware.py` importuješ z is-core. Ono by to asi spíš mělo být v is-core než v chamber.